src/wallet/wallet.cpp

```diff
@@ -1916,47 +1918,29 @@ bool CWalletTx::IsEquivalentTo(const CWalletTx& _tx) const
         return CTransaction(tx1) == CTransaction(tx2);
 }
 
-// Rebroadcast transactions from the wallet. We do this on a random timer
-// to slightly obfuscate which transactions come from our wallet.
-//
-// Ideally, we'd only resend transactions that we think should have been
-// mined in the most recent block. Any transaction that wasn't in the top
-// blockweight of transactions in the mempool shouldn't have been mined,
-// and so is probably just sitting in the mempool waiting to be confirmed.
-// Rebroadcasting does nothing to speed up confirmation and only damages
-// privacy.
+// Once a day, resbumit all wallet transactions to the node,
+// in case it has been dropped from your mempool.
 void CWallet::ResendWalletTransactions()
```

Comment:
Perhaps rename this because now we're not resending?

Reply:
we're resending (or attempting) to the mempool. I could rename to `ResubmitWalletTransactions`, but that doesn't seem much different?